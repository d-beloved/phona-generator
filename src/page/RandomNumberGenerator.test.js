import React from 'react';
import { shallow } from 'enzyme';

import RandomGenerator from './RandomNumberGenerator';

describe('Random phone number Generator', () => {
  it('should render correctly in "debug" mode', () => {

    const component = shallow(<RandomGenerator debug />);

    expect(component).toMatchSnapshot();
  });
});
