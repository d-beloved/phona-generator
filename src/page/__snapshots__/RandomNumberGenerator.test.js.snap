// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Random phone number Generator should render correctly in "debug" mode 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
      Phone Number Generator
    </p>
  </header>
  <div
    className="app-body"
  >
    <form
      className="container"
    >
      <button
        className="generateButton"
        type="button"
      >
        Generate Phone numbers
      </button>
      <div>
        <select
          className="selectBox"
        >
          <option
            value="0"
          >
            Select order
          </option>
          <option
            value="1"
          >
            Ascending order
          </option>
          <option
            value="2"
          >
            Descending order
          </option>
        </select>
      </div>
      <input
        className="number-box"
        name="input"
        placeholder="Enter a limit here..."
        type="text"
      />
      <div
        className="maxMin-instance"
      >
        <label
          className="max-num"
        >
          Max number: some numbers
        </label>
        <label
          className="min-num"
        >
          Min number: some numbers
        </label>
      </div>
    </form>
    <div
      className="number-table"
    >
      <table>
        <div
          className="scroll"
        >
          <tr>
            <th>
              S/N
            </th>
            <th>
              Phone number
            </th>
          </tr>
          <tr>
            <td>
              1
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              2
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              3
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              1
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              2
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              3
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              1
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              2
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              3
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              1
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              2
            </td>
            <td>
              07065349529
            </td>
          </tr>
          <tr>
            <td>
              3
            </td>
            <td>
              07065349529
            </td>
          </tr>
        </div>
      </table>
      <button
        className="downloadButton"
      >
        Download
      </button>
    </div>
  </div>
  <footer>
    Â© 
    2019
     
    <a
      href="https://github.com/d-beloved"
    >
      d-beloved
    </a>
  </footer>
</div>
`;
